neuralpp/util/util.py

```diff
@@ -432,7 +432,13 @@ def flatten_one_level(iterable, is_nested, get_nested):
     """
     Returns [get_nested(e) if is_nested(e) else e for e in iterable]
     """
-    return [get_nested(e) if is_nested(e) else e for e in iterable]
+    result = []
```

Comment:
It seems you fixed a bug here? I am surprised it was working before... Anyway, the docstring is now not matching the semantics of the method, so can you please change it to reflect what the method is doing?